### PR TITLE
clacheck: rename 'need-cla' label to 'hold: cla required'

### PR DESCRIPTION
**Pull request by mspncp**
It think this is all what needs to be changed for clacheck, but I can't test it.

### CODE REVIEW

**Comment by mspncp** [comment]
cc @levitte

File: clacheck/clacheck.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 FAILURE = 'failure'
 CLAFILE = "/var/cache/openssl/checkouts/omc/cladb.txt" #<EDIT>
 
+CLA_LABEL='hold: cla required'
```
Comment (levitte) [inline clacheck/clacheck.py]: Nit: perhaps do like in the rest of the file, have a little bit of spacing around the assignment operator?

File: clacheck/clacheck.py [hunk 2]
```diff
@@ -55,14 +57,14 @@ def update_status(pr, state, description):
     conn.getresponse().read()
     host,url = url_split(pr['issue_url'])
     if state == SUCCESS:
-        url = url + '/labels/need-cla'
+        url = url + '/labels/'+ urllib.quote(CLA_LABEL)
```
Comment (levitte) [inline clacheck/clacheck.py]: Same here, space before the `+` too

**Comment by levitte** [comment]
Other than those two nits, that looks right to me.

File: clacheck/clacheck.py (hunk 1 above)
Comment (mspncp) [reply clacheck/clacheck.py]: Done in ada6caa4888b12ae3aa10b97e8343a3f832df291.

File: clacheck/clacheck.py (hunk 2 above)
Comment (mspncp) [reply clacheck/clacheck.py]: Ups, didn't read far enough. Amended and force-pushed my fixup, see ac4e41a1386419e044448b019e83e575133523df.

**Comment by mspncp** [comment]
Note: When this has been merged, you might want to change the label yourself on [openssl/labels](https://github.com/openssl/openssl/labels). Instead of moving all tickets (including the closed ones!), you can just delete the [hold: cla required]  label and rename [need-cla] to [hold: cla required], both labels already have the some color.

**Comment by mspncp** [comment]
BTW: my implicit approval does not count in this repo. So it needs a second OMC review.

**Comment by t-j-h** [comment]
Nothing quite like reviewing while seated on a plane waiting for take off :-)

**Comment by mspncp** [comment]
Since the same policy as for openssl/openssl applies to also to openssl/tools and openssl/web analogously, we should add a decent subset of the new labels to those as well.

**Comment by levitte** [comment]
Merged.

73b0391902c67f9ebfcd5fa96cc10734d9631da8 clacheck: rename 'need-cla' label to 'hold: cla required'

**Comment by levitte** [comment]
Side note: I fixed all deployment keys so they were added by @openssl-machine instead of me.  Hence the push that is shown.

**Comment by mspncp** [comment]
@levitte will you rename the need-cla label, or shall I do it?

**Comment by mspncp** [comment]
> @levitte will you rename the need-cla label, or shall I do it?

I'll just do it and hope for no race condition... ;-)

**Comment by mspncp** [comment]
Done :-)

**Pull request closed without merging**